masterlist.yaml

```diff
@@ -3637,11 +3637,18 @@ plugins:
     inc:
     - 'jsawyer.esp'
     - 'Project Nevada - Rebalance.esp'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Literally Who''s Patch Emporium'
+          - '[sawyerbatty - FafnirUniqueTTWOverhaul](https://www.nexusmods.com/newvegas/mods/83169/)'
+        condition: 'active("sawyerbatty-FafnirUniqueTTWOverhaul.esp")'
```

Comment:
> **patch3rdParty**
You seem to be using **{0}**, but you have not enabled a compatibility patch for this mod. A third party patch is available here: {1}

Upon further reviewing this usage of `patch3rdParty` doesn't seem correct. This should work better:
```yaml
    msg:
      - <<: *patch3rdParty
        subs:
          - 'Fallout 3 Unique items Overhaul TTW 3.3'
          - '[Literally Who''s Patch Emporium](https://www.nexusmods.com/newvegas/mods/83169/)'
        condition: 'active("FafnirUniqueTTWOverhaul.esp") and not active("sawyerbatty-FafnirUniqueTTWOverhaul.esp")'
```